module/VuFind/tests/integration-tests/src/VuFindTest/Mink/SearchFacetsTest.php

```diff
@@ -817,7 +819,16 @@ public function testHierarchicalFacetExclude(): void
         );
         $this->clickCss($page, $this->facetExpandSelector);
         $this->clickCss($page, $this->facetSecondLevelExcludeLinkSelector);
-        $this->assertAppliedFilters($page, ['hierarchy:level1a/level2a']);
+        try {
+            $this->assertAppliedFilters($page, ['hierarchy:level1a/level2a']);
+        } catch (\Exception $e) {
+            // Sometimes the click doesn't go through possibly due to transition, so retry once:
+            $this->retryClickWithResizedWindow(
+                $this->getMinkSession(),
+                $page,
+                $this->facetSecondLevelExcludeLinkSelector
+            );
```

Comment:
Should you retry the assertion from the `try` block after the re-click to be sure the re-click had the expected effect?